src/react-queries/hooks/useQuoteEstimatedFeesQuery.ts

```diff
@@ -47,11 +67,16 @@ export const useQuoteEstimatedFeesQuery = ({
     const supportedEvmChainIds = getSupportedEvmChainIds()
     const cryptoAmount = depositAmountCryptoPrecision ?? repaymentAmountCryptoPrecision ?? '0'
     const assetId = repaymentAsset?.assetId ?? collateralAssetId
-    const quoteMemo = confirmedQuote?.quoteMemo ?? confirmedQuote?.quoteMemo ?? ''
```

Comment:
wat was this
![Added via Giphy](https://media1.giphy.com/media/xThuW4BaAA2f7nRvoc/giphy.gif?cid=c9487743fo6jcxyf5nrart428dq9mxttnjsixjlpqesdioiz&ep=v1_gifs_search&rid=giphy.gif&ct=g)

Reply:
lol, you tell me, good ser! 😂

Reply:
> lol, you tell me, good ser! 😂

don't try blaming those lines, this is showing me as the author which must be an obvious bug in git core